submissions/jeffersonMormah/database/query.sql

```diff
@@ -0,0 +1,34 @@
+-- total number of survived passengers
+SELECT COUNT(*) FROM accidents WHERE survived = 1;
+
+-- Answer = 342
+
+
+-- total number of unsurvived passengers
+SELECT COUNT(*) FROM accidents WHERE survived = 0;
+
+-- Answer = 549
+
+-- name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+SELECT passengers.firstname, passengers.lastname, passengers.sex FROM passengers
+LEFT JOIN trips ON passengers.id = trips.passenger_id
+WHERE age < 27 AND point_of_embarkation != 'S';
+
+-- total number of the passengers that embarked at Southampton and survived
+SELECT COUNT(*) FROM accidents
```

Comment:
How come you got the correct answer, but your query isn't correct.

Reply:
I apologise, that query is not correct because i altered and i did't correct before pushing. Apologies Ma'am

Reply:
Alright. well done @Jeffbuchi